NetworkingSampleApp/NetworkingSampleApp/Scenes/Upload/UploadsView.swift

```diff
@@ -53,14 +39,9 @@ struct UploadsView: View {
                 Text(viewModel.error?.localizedDescription ?? "")
             }
         )
-        .alert(
-            "Error",
-            isPresented: $formViewModel.isErrorAlertPresented,
-            actions: {},
-            message: {
-                Text(formViewModel.error?.localizedDescription ?? "")
-            }
-        )
+        .onAppear {
+            viewModel.loadTasks()
```

Comment:
We could use .task { } here and get auto cancel as loadTasks() does async work anyways.